src/Annotator.scss

```diff
@@ -448,4 +448,94 @@
     pointer-events: none;
 }
 
+//ACF
+.ba .bcs-comment-input-container {
```

Comment:
Might be worth renaming the prefix so that you don't have bcs markup in your actual components

Reply:
https://github.com/box/box-annotations/pull/224/files#diff-a3040876b949087d1140b33fd14109deR1